arangod/V8Server/v8-general-graph.cpp

```diff
@@ -298,6 +271,19 @@ static void JS_AddEdgeDefinitions(v8::FunctionCallbackInfo<v8::Value> const& arg
   VPackBuilder edgeDefinition;
   TRI_V8ToVPack(isolate, edgeDefinition, args[1], false);
 
+  VPackBuilder options;
+  if (args.Length() == 3) {
+    // We have options
+    TRI_V8ToVPack(isolate, options, args[2], false);
+  }
+  if (!options.slice().isObject()) {
+    options.clear();
+    // Fake empty options, silently ignore errors for now.
+    // Empty Options.
+    options.openObject();
+    options.close();
+  }
```

Comment:
This doesn't seem polite. Shouldn't we rather report a meaningful error?

Reply:
Agreed, but currently this is the way it is handled like everywhere else. 
This should be refactored generally in another PR.